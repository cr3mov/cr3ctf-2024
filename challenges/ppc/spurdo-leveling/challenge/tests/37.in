133973126274107027 960703050488618356
2 5653
1 7067
