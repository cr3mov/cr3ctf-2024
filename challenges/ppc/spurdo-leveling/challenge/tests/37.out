116968014178624 0 116968014178624
